web/docs/advanced/accessing-app-config.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Accessing the configuration 
+---
+
+Whenever you start a Waps app, you are starting two processes.
```

Comment:
Might I suggest using `app` instead of `process` to keep it a bit less technical i.e. " you are starting two apps"

Reply:
Are you sure?

The article mainly talks about environment variables. I think processes make more sense at this level of abstraction

Reply:
It sounded very technical for me "we are starting two processes", I was expected to hear the word multi-threading next 😄 That's why I wanted to tone it down a bit to used "app", at least in the intro section. But maybe I'm overreacting, decide what feels the best, I'm okay with either option.

Reply:
I decided to go with process because "app" would mean two different things in that sentence: "When you start a Wasp *app*, you are starting two _apps_."